articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/machine-learning-and-deep-learning/
+title: Machine Learning and Deep Learning
+description:
+author: james-kimani
+date: 2020-10-28T00:00:00-17:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+
+  - url: /engineering-education/machine-learning-and-deep-learning/hero.jpg
+    alt:
+---
+Technology is growing at an alarming pace in recent years. Due to this, it is now easy to get lost in the ruckus of all technological terms that keep popping up every day.
+<!--more-->
+### Introduction
+Machine learning and deep learning are among the fastest-growing applications. Artificial Intelligence ([AI](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp)) is one of the fast-growing technologies that has significantly contributed to the growth of machine and deep learning.
+
+The AI incorporates and helps power the intelligence that is used to enhance decision making within both applications. These applications have contributed significantly in solving problems that previously affected performances in some applications.  
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Alexa and Siri are [virtual assistant](https://www.investopedia.com/terms/v/virtual-assistant.asp) that make extensive use of machine learning and are widely used today.
+
+Machine learning and deep learning have attracted significant attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. By the end of the article the reader will have a deeper understanding on how these applications differ and enhances AI functionalities.
+
+### Machine Learning and Artificial Intelligence
+Machine learning is a [subset](https://en.wikipedia.org/wiki/Subset) of AI that allows computer algorithms to make accurate data outcomes through interpretations.
+
+Machine learning allows the use the AI towards automating several functionalities used by machine learning. Besides this, the AI enhances neuroscience that emulates human intelligence, an aspect that machine learning lacks.
```

Comment:
This sentence is a bit difficult to understand. 
"Machine learning allows the use the AI towards automating several functionalities used by machine learning. Besides this, the AI enhances neuroscience that emulates human intelligence, an aspect that machine learning lacks."

Reply:
noted, I have made a simpler elaboration on the point.